cli/templates/JavaScript/package.json

```diff
@@ -8,7 +8,9 @@
     "npm": "<7.0.0"
   },
   "dependencies": {
+    // <<protectedData>>
     "@iexec/dataprotector-deserializer": "^0.1.1",
+    // <</protectedData>>
```

Comment:
wrap `@iexec/dataprotector-deserializer` in feature tag to avoid installation when not needed, this require to apply feature tag post treatment on this file... it too bad `package.json` are not `jsonc`